src/utility/utility.ts

```diff
@@ -19,15 +19,14 @@ export class Utility {
         return true;
     }
 
-    public static copy<T extends Object>(target: T, source: any): T {
+    public static copy<T extends object>(target: T, source: any): T {
         Object.keys(target).forEach(key => {
```

Comment:
where is 'Object' from?

Reply:
https://stackoverflow.com/questions/49464634/difference-between-object-and-object-in-typescript